src/sass/patterns/_tables.scss

```diff
@@ -0,0 +1,29 @@
+// Tables
+// ======
+
+.minimal-th {
```

Comment:
This does look nicer than the default – is there SLDS precedent, or are we intentionally going off-script here?

Reply:
As far as I can tell we're just going off-script (cf https://www.lightningdesignsystem.com/components/data-tables/). @stacyk or @SondraE thoughts or reasoning?

Reply:
I believe this was designed before we knew we were using the datatable component. We originally had [expandable sections in a list](https://xd.adobe.com/view/755c3a4f-db06-4098-47a4-de2b8684fb8b-036d/) that didn't have a header row. Once it evolved into a table, I think either Sondra or I added the headers in bold but didn't include the header bg row. I do think it was a happy accident, but if it is a problem, we can remove this minimal-th to match more with SLDS

Reply:
My instinct when using SLDS is generally to align whenever possible, unless that would look visually broken for some reason.

Reply:
@mirisuzanne @stacyk Changed to use standard SLDS table. Please re-review and cleanup as needed.

![Screenshot 2019-08-12 17 21 42](https://user-images.githubusercontent.com/552316/62899394-aae3b480-bd25-11e9-8814-60f9e9954d65.png)